tests/test_encryption.py

```
@@ -16,7 +21,7 @@ def test_decrypt():
 
 def test_encrypt_decrypt():
     passphrase = "correcthorsebatterystaple"
-    encryptor = AESEncryptor(passphrase, iterations=1000)
+    encryptor = Encryptor(passphrase=passphrase, iterations=1000) # Passando passphrase
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)